controls/inetutils_exists.rb

```diff
@@ -0,0 +1,23 @@
+title 'Tests to confirm inetutils exists'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'inetutils')
+plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
```

Comment:
Would be good to confirm this command was successful before using it in subsequent tests.

Reply:
Done, thanks.